lib/HooksHandler.php

```diff
@@ -262,7 +262,7 @@ public function savePasswordForCreatedUser(GenericEvent $event) {
 	/**
 	 * Flags the session to require a password change
 	 */
-	protected function forcePasswordChange($firstLogin = null, IUser $user = null) {
+	protected function forcePasswordChange(?bool $firstLogin, ?IUser $user) {
```

Comment:
```suggestion
	protected function forcePasswordChange(?bool $firstLogin = null, ?IUser $user = null) {
```

We can leave `= null` so that the interface of this method remains the same - users cam call it without specifying parameters, if they like.